src/attributes/classes.js

```diff
@@ -23,9 +39,7 @@ jQuery.fn.extend( {
 			} );
 		}
 
-		if ( typeof value === "string" && value ) {
-			classes = value.match( rnothtmlwhite ) || [];
```

Comment:
Thanks, @dmethvin. The markers have been removed. I am using VS Code. and for some reason the tab size in the code seems correct and uniform throughout, JShint and grunt passes. However, after the push, the uploaded file shows the tab size to be exaggerated. Any idea how I can resolve this issue?